translations/it.js

```diff
@@ -124,7 +124,7 @@ var it = {
 
 "alertNonDescribedVersion": "Sto usando la versione non descritta di questo video",
 
-"fallbackError1": "Spiacente il tuo browser non � in grado di riprodurre questo",
+"fallbackError1": "Spiacente il tuo browser non � in grado di riprodurre questo",
```

Comment:
hmm what the hey..? 🤔

Reply:
Looks like something transformed it, is there a character set setting somewhere?

Reply:
Fixed - that file was updated recently in the main repo so I updated it also.